src/sc/proofverifier.cpp

```diff
@@ -33,7 +33,13 @@ std::string ProofVerificationResultToString(ProofVerificationResult res)
  * 
  * @return CCertProofVerifierInput The structure containing all the data needed for the proof verification of the certificate.
  */
-CCertProofVerifierInput CScProofVerifier::CertificateToVerifierItem(const CScCertificate& certificate, const Sidechain::ScFixedParameters& scFixedParams, CNode* pfrom)
+#ifdef BITCOIN_TX
+CCertProofVerifierInput CScProofVerifier::CertificateToVerifierItem(const CScCertificate& certificate, const Sidechain::ScFixedParameters& scFixedParams, CNode* pfrom, const CCoinsViewCache* view)
+{
+    return CCertProofVerifierInput();
+}
+#else
+CCertProofVerifierInput CScProofVerifier::CertificateToVerifierItem(const CScCertificate& certificate, const Sidechain::ScFixedParameters& scFixedParams, CNode* pfrom, const CCoinsViewCache* view)
```

Comment:
Please insert pre-processor directives inside function body when function signature is identical

Reply:
The pattern I used is quite common all over our codebase. I'd rather keep it consistent for now, and refactor all these cases in another PR if we want to.

Reply:
Fair enough. Thank you.